FordLabs/retroquest
# Add the ability to delete one archived board at a time

## What was done
- [x] Add the ability to delete one archived board at a time by adding a delete button to each archived board.
- [x] Updated the thoughts archives page to show 20 boards per page vs 6. 

### Demo
<img width="970" alt="Screen Shot 2022-10-26 at 11 45 52 AM" src="https://user-images.githubusercontent.com/17144844/198073198-28caf06b-923e-4475-bbe5-dd5ad2339e80.png">

## Testing Instructions
1) Go to Archives>Thought Archives (if no archived boards there, create some thoughts and archive a few boards)
2) Click on the delete button on one of the created boards. Ensure a confirmation modal pops up.
3) Click cancel and ensure nothing happens.
4) Click the delete button again and this time click "Yes, Delete". Ensure that the archived board gets deleted by ensuring the board is no longer on the page.
